pyiron-experimental/environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+- conda-forge
+dependencies:
+- temmeta = 0.0.6
```

Comment:
```suggestion
- temmeta =0.0.6
- pyprismatic =1.2.1 
- match-series =1.0.0 
- pyxem =0.12.2
```

Reply:
The spaces were for better readability. I will remove these.